packages/synapse-react-client/src/lib/containers/auth/LoginFlowBackButton.tsx

```diff
@@ -39,6 +39,7 @@ export default function LoginFlowBackButton(props: LoginFormBackButtonProps) {
   }
   return (
     <IconButton
+      className="login-flow-back-button"
```

Comment:
Allow styling of the back button, if needed

Reply:
Since you reference this in other modules, consider extracting to an exported constant

```tsx
export const LOGIN_BACK_BUTTON_CLASS_NAME="login-flow-back-button"
```

in other file:

```tsx
        <Box
          sx={{
            py: 10,
            px: 8,
            height: '100%',
            position: 'relative',
            [`.${LOGIN_BACK_BUTTON_CLASS_NAME}`]: backButtonSx,
          }}
```